episodes/medical_imaging.md

```diff
@@ -100,6 +162,7 @@ For example, while typical radiological images have settled in how they are reco
 
 - Each imaging modality provides distinct sets of information
 - In computational imaging, images are essentially arrays, although embedded in additional data structures
+- Many images we may get e.g. MRIs and CTs have already been processed with some algorithms to make them human readable
 - Research should be thoughtfully designed, taking into account the constraints and capabilities inherent in human capacities
```

Comment:
Not sure if you touch upon this in the episode

Reply:
This is the whole things of the CT being processed and the MRIs being processed (from k-space). I guess I need to make this clearer